packages/blocks/src/render-blocks.js

```diff
@@ -95,7 +96,16 @@ const appendInnerBlocks = ( blockName, elements, innerBlocks ) => {
 
 		// 	container.innerBlocks = innerBlocks;
 		// 	return;
```

Comment:
Nit: could we remove this commented code while we're at it? It was here only as a reference but now that we have a working example, I think we can just get rid of it.

Reply:
Sure!